test/unit/services/upload.spec.js

```diff
@@ -79,4 +87,60 @@ describe('uploadService', () => {
         expect(err).toBe('error')
       }
     })
+  it('should update a file in Azure Blob Storage successfully', async () => {
+    const file = {
```

Comment:
can you move file above to and don`t define it in multiple tests?

Reply:
fixed